nav2_msgs/action/NavigateThroughPoses.action

```diff
@@ -4,6 +4,7 @@ string behavior_tree
 ---
 #result definition
 std_msgs/Empty result
+int16 error_code
```

Comment:
waypoint follower too? Also, you need to extract that from the WPF result if there's a failure

Reply:
(basically just taking the result from NavToPose and passing it through to the WPF client, its a 1-liner)